pages/linux/wl-paste.md

```diff
@@ -0,0 +1,25 @@
+# wl-paste
+
+> Tool to access data stored in the clipboard for Wayland.
+> See also: `wl-copy`.
+> More information: <https://manned.org/wl-paste>.
```

Comment:
We should use the official homepage here. This probably should also be updated for `wl-copy`.
```suggestion
> More information: <https://github.com/bugaevc/wl-clipboard>.
```